course6/toyapp_mikhail/.gitignore

```diff
@@ -0,0 +1,2 @@
+build
```

Comment:
1 added to gitignore for c)
2 I have multiple of them inside build folder
3 I delete entire build folder usually, then do 
   mkdir build/build-debug
   cd build/build-debug
   CMake ./../.. 
   MSBuild ALL_TRAGETS
4 Sure I can use folder completely outside of whole git repo but it will be more burden with folders, as my sources not in root of git repo.
  instead of:
  CMake ./../my-repo
  it will be
  CMake ./../../../c-excersises/cource6/toyapp_mikhail
I will create mycmake.bat to make it easier